<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 1,
        stop: 0 #3a3a3a,  /* Start color (red) */
        stop: 1 #212121   /* End color (blue) */
    );
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="usrname_label">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_edit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: #282828;
	color: white;


}</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pass_label">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass_edit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: #282828;
	color: white;


}</string>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="login_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #282828;
 	color: white;
	border-radius: 5%;
	border: none;
	height: 30%;
}
QPushButton:hover {
	background-color: #212121;

}
QPushButton:pressed {
	background-color: #191919;
	padding-left: 52%;
}

</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="switch_to_log_btn_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #282828;
 	color: white;
	border-radius: 5%;
	border: none;
	height: 30%;
}
QPushButton:hover {
	background-color: #212121;

}
QPushButton:pressed {
	background-color: #191919;
	padding-left: 52%;
}

</string>
           </property>
           <property name="text">
            <string>Sign up?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="reg_name_edit">
           <property name="styleSheet">
            <string notr="true">background-color: #282828;
color: white;</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: white;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="reg_pass_edit">
           <property name="styleSheet">
            <string notr="true">background-color: #282828;
color: white;</string>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reg_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #282828;
 	color: white;
	border-radius: 5%;
	border: none;
	height: 30%;
}
QPushButton:hover {
	background-color: #212121;

}
QPushButton:pressed {
	background-color: #191919;
	padding-left: 52%;
}

</string>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="switch_to_log_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #282828;
 	color: white;
	border-radius: 5%;
	border: none;
	height: 30%;
}
QPushButton:hover {
	background-color: #212121;

}
QPushButton:pressed {
	background-color: #191919;
	padding-left: 52%;
}

</string>
           </property>
           <property name="text">
            <string>Already registered?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
